js/scripts/util.js

```diff
@@ -380,4 +380,17 @@ const EVENT_HANDLERS = {
                 GAME.user.continuingConversation = false;
         }
     },
-};
\ No newline at end of file
+};
+
+
+// custom CTX functions
+
+const roundRect = function(x, y, width, height, radius) {
```

Comment:
This is super cool but how does this work? Is this something built into JS or is this a functionality similar to JSON Objects that I am unfamiliar with?

Reply:
Still trying to work out the bugs with that. I was trying to use CTX to trace out a rectangle with rounded edges.